src/cascadia/TerminalApp/ActionArgs.h

```diff
@@ -116,8 +119,64 @@ namespace winrt::TerminalApp::implementation
             {
                 args->_SingleLine = singleLine.asBool();
             }
+            if (auto copyFormatting{ json[JsonKey(CopyFormattingKey)] })
+            {
+                args->_CopyFormatting = _ParseCopyFormatting(copyFormatting);
+            }
             return { *args, {} };
         }
+
+        // accepted copyFormatting values
+        static constexpr std::string_view PlainKey{ "plain" };
+        static constexpr std::string_view HtmlKey{ "html" };
+        static constexpr std::string_view RtfKey{ "rtf" };
+
+        // Method Description:
+        // - Helper function for converting the user-specified copyFormatting value(s)
+        //   to a set of CopyFormat enum values
+        // Arguments:
+        // - json: The string value from the settings file to parse
+        // Return Value:
+        // - The serialized enum values which map to the bool or array of strings provided by the user
+        // - -1 represents parsing failure or null. Behavior falls back to the global setting.
+        static int _ParseCopyFormatting(const Json::Value& json) noexcept

```

Comment:
One thing I hate is that there _is_ a minor difference between the ActionArg parser and the GlobalAppSetting parser. When set to `null` or an invalid value...
- ActionArg parser: return -1 to signify that we fallback to the global value
- GlobalAppSetting parser: return "all"

Any ideas on a logical place to move all of this? This is also frustrating because it'll _really_ be impacted by #6590, so it might just make more sense to wait until that gets merged?